### PR TITLE
Added route-behavior to article-page

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | build ontop #122 
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR adds the route to article-page.

#### TODO

- [x] Tests

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -28,6 +28,14 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder();
         $treeBuilder->root('sulu_article')
             ->children()
+                ->arrayNode('pages')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('route_schema')
+                            ->defaultValue('/{translator.trans("page")}-{object.getPageNumber()}')
```
Comment (alexander-schranz) [inline DependencyInjection/Configuration.php]: would prefer to configure this also in the sulu_route

File: DependencyInjection/SuluArticleExtension.php [hunk 2]
```diff
@@ -94,6 +94,23 @@ public function prepend(ContainerBuilder $container)
...
 
+        if ($container->hasExtension('sulu_route')) {
+            $container->prependExtensionConfig(
+                'sulu_route',
+                [
+                    'mappings' => [
+                        ArticlePageDocument::class => [
+                            'generator' => 'article_page',
+                            'options' => [
+                                'route_schema' => '/{translator.trans("page")}-{object.getPageNumber()}',
```
Comment (alexander-schranz) [inline DependencyInjection/SuluArticleExtension.php]: would call the parameter just `schema` as we are in the route definition

Comment (wachterjohannes) [inline DependencyInjection/SuluArticleExtension.php]: there it is also `route_schema`

Comment (alexander-schranz) [inline DependencyInjection/SuluArticleExtension.php]: ok

**Pull request merged**